blogs/posts/2024-02-11_sankey_plot.qmd

```diff
@@ -0,0 +1,565 @@
+---
+title: "Visualising participant recruitment with Sankey plots"
+author:
+  - Craig Parylo
+date: "2024-02-11"
+categories: [learning, tutorial, visualisation]
+reference-location: margin
+citation-location: margin
+---
+
+```{r}
+#| code-fold: true
+#| warning: false
+#| message: false
+#| output: false
+
+# libraries
+library(tidyverse)       # 'tidy' data wrangling
+library(plotly)          # sankey visualisation framework
+library(reactable)       # viewing interactive datatables
+library(glue)            # concatenating strings
+library(tidygraph)       # api for graph / network manipulation
+library(scales)          # used for colour transformation
+```
+
+# Introduction
+
+Sankey diagrams are great tools to visualise flows through a system. They show the flow / conversion rate after each step of a process and the width of the arrows is proportional to the flow / conversion rate.
+
+I'm working on a project to evaluate the effectiveness of a risk screening programme for people aged between 55-74 years and a history of smoking[^1]. One report requirement is to be able to visualise how people have engaged with the programme, from recruitment, attendance at appointments, their outcome from risk assessment, low-dose CT scans and will eventually cover the impact of the screening on early detection of those diagnosed with lung cancer.
+
+[^1]: For more background on the TLHC programme see <https://www.england.nhs.uk/contact-us/privacy-notice/how-we-use-your-information/our-services/evaluation-of-the-targeted-lung-health-check-programme/>
+
+The use of Sankey plots has helped the programme team to better understand patient flows through the pathway, points of drop-off and differences between different intervention models.
+
+Here is a link to the final Sankey diagram for the full TLHC programme: <https://connect.strategyunitwm.nhs.uk/content/3f114580-bcb7-42d8-82a5-b607baf96da5/>
```

Comment:
I assume this is available to the general public?

Reply:
I'm awaiting permission to use in a public-facing site. In the meantime I've added a stored copy of the final chart to introduce the reader to how Sankeys are used in this work.